pkg/handlers/v1/eni.go

```diff
@@ -0,0 +1,115 @@
+package v1
+
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+)
+
+const elbManaged = "amazon-elb"
```

Comment:
nit: rename to requesterManaged

Reply:
Unfortunately using requesterManaged ended up looking confusing next to the actual field called RequesterManaged. I went with elbRequested as a compromise since I agree elbManaged didn't seem quite right.

Reply:
Sounds good